### ISSUE TITLE
Is the in-node concurrency supported now?

### ISSUE THREAD

**Issue by zhang943** [lf-edge/ekuiper#3343]
> 1. concurrency param becomes useless, will add it back later with in-node concurrency

PR #2569 removed node concurrency, is the in-node concurrency supported now?

**Comment by ngjaying** [comment]
Yes, some node can support concurrency like encode/decode, sink transform, compress/decompress. These nodes will spawn the defined number of workers but still deliver result in order.
